cachix-api/ChangeLog.md

```diff
@@ -7,6 +7,19 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [0.1.0.2] - 2018-09-27
+
+### Changed
+
+- #133 Address all GHC warnings @domenkozar
```

Comment:
I don't think refactoring is relevant unless it affects public interfaces, in which case it is the public interface that is relevant, not the refactoring. That doesn't apply here.